### PR TITLE
Task/tsn 918 memory pages

### PR DESCRIPTION
**Pull request by tmassuia-sw**
On this branch: 

- Memories page almost done
To do: 
Link to memories query
details on dropdown menu

### CODE REVIEW

File: packages/apps/web/src/components/Layout/Home/Home.js [hunk 1]
```diff
@@ -102,6 +102,15 @@ const Home = () => {
 											text={t('HOME.GO_TO_FAMILY_TREE')}
 										/>
 									</FlexContainer>
+									<FlexContainer direction={FLEX_DIR.ROW} justify={FLEX_JUSTIFY.END} align={FLEX_ALIGN.CENTER}>
+										<Button
+											buttonStyle={BUTTON_STYLE.CLEAR}
+											color={BUTTON_COLOR.MAIN} size={BUTTON_SIZE.SMALL}
+											click={() => routingContext.changeRoute('/memories/')}
```
Comment (mrilo-sw) [inline packages/apps/web/src/components/Layout/Home/Home.js]: add +profileid to change route

File: packages/apps/web/src/components/Layout/Home/Home.js [hunk 2]
```diff
@@ -102,6 +102,15 @@ const Home = () => {
...
+											//											text={t('HOME.GO_TO_FAMILY_TREE')}
+											text={'go to memories'}
```
Comment (mrilo-sw) [inline packages/apps/web/src/components/Layout/Home/Home.js]: add i18n

File: packages/apps/web/src/components/Layout/Home/Home.js (hunk 1 above)
Comment (tmassuia-sw) [reply packages/apps/web/src/components/Layout/Home/Home.js]: This route should redirect to the memories list.
In detail, we will redirect with memory id.

File: packages/apps/web/src/components/Layout/Home/Home.js (hunk 2 above)
Comment (tmassuia-sw) [reply packages/apps/web/src/components/Layout/Home/Home.js]: This is temporary. Daniel's should bring us a special way to display it and then we'll add a translation

**Pull request closed without merging**